exemplar/src/main/java/com/squareup/exemplar/ExemplarJavaModule.java

```diff
@@ -10,5 +11,6 @@
     install(WebActionModule.create(HelloJavaAction.class));
     install(WebActionModule.create(InternalErrorAction.class));
     install(WebActionModule.create(NotFoundAction.class));
+    install(WebActionModule.create(MetricsAction.class));
```

Comment:
Same here, should be one line up.

Reply:
Done. BTW the java module is missing the liveness and readiness actions.